packages/gettext.rb

```diff
@@ -23,15 +23,19 @@ class Gettext < Package
      x86_64: '4ada2428329f3fc4098b4fab8cc81d816028a7ac283c359c86fdc1d569a52af3'
   })
 
+  depends_on 'acl' # R
+  depends_on 'gcc' # R
+  depends_on 'glibc' # R
   depends_on 'jdk8' => :build
```

Comment:
Does this build with `openjdk8`?  Update if so?

Reply:
Possibly? But checking to see if packages that depend upon `jdk8` also build with `openjdk8` is outside the scope of this PR. I opened an issue for that: https://github.com/chromebrew/chromebrew/issues/7476